index.html

```diff
@@ -69,102 +71,89 @@
-                "
-              ></div>
-            </div>
-            <div
-              class="pl-3 border-l border-solid border-gray-700 h-full flex items-center"
-            >
-              <a
-                class="bg-mb-purple-500 hover:bg-mb-purple-700 hover:text-white text-white text-xs font-bold py-1 px-4 rounded transition-all duration-200"
-                href="https://account.mapbox.com/auth/signup/"
-              >
-                Build with Mapbox
-              </a>
-            </div>
+                "></div>
+          </div>
+          <div class="pl-3 border-l border-solid border-gray-700 h-full flex items-center">
+            <a class="bg-mb-purple-500 hover:bg-mb-purple-700 hover:text-white text-white text-xs font-bold py-1 px-4 rounded transition-all duration-200"
+              href="https://account.mapbox.com/auth/signup/">
+              Build with Mapbox
+            </a>
           </div>
         </div>
       </div>
     </div>
+  </div>
+  <script>
+    if (self == top) {
+      document.getElementsByTagName("body")[0].style.display = "block";
+    } else {
+      top.location = self.location;
+    }
+  </script>
+  <script src="dist/delegate.js"></script>
+  <script src="dist/lib.js"></script>
+  <script src="dist/site.js"></script>
+  <script type="text/javascript">
+    var _gauges = _gauges || [];
+    (function (i, s, o, g, r, a, m) {
+      i["GoogleAnalyticsObject"] = r;
+      (i[r] =
+        i[r] ||
+        function () {
+          (i[r].q = i[r].q || []).push(arguments);
+        }),
+        (i[r].l = 1 * new Date());
+      (a = s.createElement(o)), (m = s.getElementsByTagName(o)[0]);
+      a.async = 1;
+      a.src = g;
+      m.parentNode.insertBefore(a, m);
+    })(
+      window,
+      document,
+      "script",
+      "https://www.google-analytics.com/analytics.js",
+      "ga"
+    );
+    ga("create", "UA-12158002-18", "auto");
```

Comment:
We own this tag, right? Or does Tom still hold it?

Reply:
We own it.